--- smpeg-xmms-0.3.5/src.Makefile.am~	Fri Jan 11 16:12:45 2002
+++ smpeg-xmms-0.3.5/src/Makefile.am	Fri Jan 11 16:12:45 2002
@@ -1,10 +1,10 @@
 lib_LTLIBRARIES = libsmpeg_xmms.la
 
-libdir = @XMMS_INPUT_PLUGIN_DIR@
+libdir = $(XMMS_INPUT_PLUGIN_DIR)
 
 libsmpeg_xmms_la_LDFLAGS = -module -avoid-version
 
-INCLUDES += @XMMS_CFLAGS@ @SDL_CFLAGS@ @SMPEG_CFLAGS@
-libsmpeg_xmms_la_LIBADD += @SDL_LIBS@ @SMPEG_LIBS@
+INCLUDES = $(XMMS_CFLAGS) $(SDL_CFLAGS) $(SMPEG_CFLAGS)
+libsmpeg_xmms_la_LIBADD = $(SDL_LIBS) $(SMPEG_LIBS)
 
 libsmpeg_xmms_la_SOURCES = smpeg-xmms.c smpeg-xmms.h configure.c
